kratos/python/add_variable_utils_to_python.cpp

```diff
@@ -455,7 +455,6 @@ void AddVariableUtilsToPython(pybind11::module &m)
         .def("AddDof", &VariableUtils::AddDof<Variable<double>>)
         .def("AddDof", &VariableUtils::AddDofWithReaction<Variable<double>>)
         .def("CheckVariableKeys", &VariableUtils::CheckVariableKeys)
```

Comment:
We could probably remove this too. Am I wrong?

Reply:
this does some more stuff, it also checks registration of the Variables

but it is also not used anywhere so I guess we could also remove it

I never had any error related to this anyway :man_shrugging: